include/nuttx/mtd/configdata.h

```diff
@@ -120,6 +120,27 @@ extern "C"
 struct mtd_dev_s;
 int mtdconfig_register(FAR struct mtd_dev_s *mtd);
 
+/****************************************************************************
+ * Name: mtdnvs_register
+ *
+ * Description:
+ *   This function binds an instance of an MTD device to the /dev/config
+ *   device.
+ *
+ *   When this function is called, the MTD device pass in should already
+ *   be initialized appropriately to access the physical device or partition.
+ *
+ * Input Parameters:
+ *   mtd - Pointer to the MTD device to bind with the /dev/config device
+ *
+ * Returned Value:
+ *   Zero on success; a negated errno value on failure.
+
+ *
+ ****************************************************************************/
+
+int mtdnvs_register(FAR struct mtd_dev_s *mtd);
```

Comment:
let's use the same register function mtdconfig_register?

Reply:
Isn't it confusing to use the same function name? After all, nvs is a different method. I don't see the need to make them look the same.

Reply:
The NVS is just a different backend.